### PR TITLE
Add how-to-use page + placeholders in search form.

### PR DESCRIPTION
**Pull request by KDercksen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,12 +17,13 @@
 <body>
 	<a href="https://github.com/dopefishh/cloogle">
 			<img src="logo.jpg" alt="follow link for the sourcecode" />
-	</a>
+    </a><br />
+    <a href="help.html">How to use</a>
```
Comment (camilstaps) [inline index.html]: Please use tabs instead of spaces

**Comment by dopefishh** [comment]
wut, that `help.html`...

**Comment by camilstaps** [comment]
Thanks. A few suggestions:
- I don't think we need a separate page for this. Just hide it by default and show/hide with javascript.
- It seems there are several unnecessary things in help.html: all the javascript, the icon, almost all the CSS. I'd suggest working without framework, it's really overkill here. We don't need half an MB for a help page...

**Comment by KDercksen** [comment]
Luckily that giant page was not manual work! :smile:

File: index.html [hunk 2]
```diff
@@ -5,6 +5,15 @@
 	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<script src="api.js" type="text/javascript" defer="defer"></script>
+	<script type="text/javascript">
+		function toggle(name) {
+            var e = document.getElementById(name);
```
Comment (camilstaps) [inline index.html]: Tabs here

**Comment by camilstaps** [comment]
This is now live. Thanks!

**Pull request merged**